### PR TITLE
Blog App: Add devise

### PR DESCRIPTION
**Pull request by sja-thedude**
- Removed current_user method in ApplicationController.
- Install and set up devise.
Registers a new user.
User logs in with a combination of email and password.
Hashed passwords should be stored in the database.
Asks for confirmation of email.
Resets password.
Screenshot 2022-03-01 at 12 18 33 PM

Note-: Check the image above which confirms the email link in the terminal

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (t-yanick) [inline app/models/user.rb]: - [x] Please in order for the user to confirm his/her email, `:confirmable` has to be enabled in the `User model`, please kindly fix that. See screenshot below for clarifications and kindly fix that.

confirmation

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @sja-thedude, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (sja-thedude) [reply app/models/user.rb]: Noted sir, Added confirmable.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi  :wave:   :wave: @sja-thedude  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your email has confirmation as requested by the previous reviewer :heavy_check_mark: 
- Your README file looks very professional :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**